apps/website/foundation/icons/usage.md

```diff
@@ -37,23 +37,17 @@ The default rotation for every shape in Clarity Icons is 0 degrees. With directi
 <doc-code>
 
 ```html
-<!--A. SETTING THE ROTATION DIRECTION THROUGH CLR-ICON SHAPE ATTRIBUTE-->
-<clr-icon shape="caret up"></clr-icon>
-<clr-icon shape="caret right"></clr-icon>
-<clr-icon shape="caret down"></clr-icon>
-<clr-icon shape="caret left"></clr-icon>
-
-<!--B. SETTING THE ROTATION DIRECTION THROUGH CLR-ICON DIR ATTRIBUTE-->
-<clr-icon shape="caret" dir="up"></clr-icon>
-<clr-icon shape="caret" dir="right"></clr-icon>
-<clr-icon shape="caret" dir="down"></clr-icon>
-<clr-icon shape="caret" dir="left"></clr-icon>
-
-<!--C. SETTING THE ROTATION DIRECTION IN STYLE ATTRIBUTE-->
-<clr-icon shape="caret" style="transform: rotate(0deg);"></clr-icon>
-<clr-icon shape="caret" style="transform: rotate(90deg);"></clr-icon>
-<clr-icon shape="caret" style="transform: rotate(180deg);"></clr-icon>
-<clr-icon shape="caret" style="transform: rotate(270deg);"></clr-icon>
+<!--A. SETTING THE ROTATION DIRECTION THROUGH CLR-ICON DIRECTION ATTRIBUTE-->
```

Comment:
This comment is visible in examples, right? Then, we might want to change CLR to CDS.